Gems/AWSMetrics/cdk/api_spec.json

```diff
@@ -3,7 +3,7 @@
     "info": {
         "title": "AWSMetricsServiceApi",
         "description": "Service API for the data analytics pipeline defined by the AWS Metrics Gem",
-        "version": "1.0.0"
+        "version": "1.0.1"
```

Comment:
Minor: it should be a minor or patch? it doesn't feel like a bug fix, or course it is not a feature either.

Reply:
I would say it's a patch since we don't introduce any new functionalities.